spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -1844,112 +1844,104 @@ it to be applied to that entity or mapped superclass and its subclasses.
-validation is in effect. The values of the `validation-mode` element are
-`AUTO`, `CALLBACK`, `NONE`. The default validation mode is `AUTO`.
-
-If the application creates the entity manager
-factory using the `Persistence.createEntityManagerFactory` method, the
-validation mode can be specified using the
-`jakarta.persistence.validation.mode` map key, which will override the
-value specified (or defaulted) in the `persistence.xml` file. The map
-values for this key are _"auto"_, _"callback"_, _"none"_.
-
-If the auto validation mode is specified by
-the `validation-mode` element or the `jakarta.persistence.validation.mode`
-property, or if neither the `validation-mode` element nor the
-`jakarta.persistence.validation.mode` property is specified, and a Bean
-Validation provider is present in the environment, the persistence
-provider must perform the automatic validation of entities as described
-in <<a2380>>. If no Bean Validation provider is
-present in the environment, no lifecycle event validation takes place.
-
-If the callback validation mode is specified
-by the `validation-mode` element or the
-`jakarta.persistence.validation.mode` property, the persistence provider
-must perform the lifecycle event validation as described in <<a2380>>.
-It is an error if there is no Bean Validation
-provider present in the environment, and the provider must throw the
-`PersistenceException` if the `jakarta.persistence.validation.mode`
-property value _"callback"_ has been passed to the
-`Persistence.createEntityManagerFactory` method.
-
-If the none validation mode is specified by
-the `validation-mode` element or the `jakarta.persistence.validation.mode`
-property, the persistence provider must not perform lifecycle event
-validation.
+This specification defines the following validation modes:
+
+* `auto` -- if the Jakarta Validation _provider_ is obtainable,
+then the persistence provider must perform the automatic validation of entities
+as described in <<a2380>>.
+Otherwise, no lifecycle event validation takes place.
+This is the default validation mode.
+* `callback` -- the persistence provider must perform the automatic validation of entities
+as described in <<a2380>>. If no Jakarta Validation _provider_ is available,
+the persistence provider must throw the `PersistenceException`.
+* `none` -- the persistence provider must not perform any lifecycle event validation.
+
+[NOTE]
+====
+The Jakarta Validation _provider_ is obtained via the `jakarta.validation.ValidatorFactory` (<<a2394>>).
+====
+
+The validation mode may be specified via the `validation-mode` element
+of the `persistence.xml` file.
+Allowed values for the `validation-mode` element are `AUTO`, `CALLBACK`, `NONE`.
+
+The validation mode value derived from the `persistence.xml`
+may be overridden by the `jakarta.persistence.validation.mode` configuration property
+(e.g. passed to the  `Persistence.createEntityManagerFactory` factory method).
```

Comment:
It's always better to given an explicit list of possibilities than leave an "e.g." hanging in the air. I believe that in this case, there are only two possibilities.

```suggestion
The validation mode value specified in `persistence.xml` may be overridden by 
the configuration property `jakarta.persistence.validation.mode` which may be
passed to `Persistence.createEntityManagerFactory` or 
`PersistenceProvider.createContainerEntityManagerFactory`, as defined in
<<a13443>> and <<a12802>>.
```

Reply:
Sure, refactoring out copypasta into a proper definition is very often better than laboriously reciting the same phrase over and over. You're allowed to add new sections.